curriculum/en/6-building-your-frontend/7-finishing-event-page.md

```diff
@@ -177,6 +177,10 @@ If the event is successfully created, we can show the user a success message and
 
 And that's it! Test out the page to see if you are able to successfully create a new event.
 
+> Note: Once you test that you have successfully created a new event and try to visit the 
```

Comment:
can we update this copy to

Note: Upon successfully creating an event, you'll see a confirmation message that includes a link to your event page. You'll be working on setting up event pages in **Section 7**. Onwards!

Reply:
also in line 189, wanna add your twitter to the list of editors?

, [Jordan Rob](link to your twitter)